### ISSUE TITLE
As an app-developer I would like to add a link to our accessibility statement (Tilgjengelighetserklæring)

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#538]
NB: This is not A & AA. I just want it to visible as it is important for the accessibility statement.
### Description

Add the link to our accessibility statement in a footer. 
![Image](https://user-images.githubusercontent.com/33222679/213448820-32284f23-05c6-4e13-901b-a35b0c129329.png)
[Figma](https://www.figma.com/file/Pvti3aRcOwH0k5Z7DrLznk/Arbeidsomr%C3%A5de%3A-Altinn-Studio-APPS?node-id=4213%3A78024&t=WPn0BkUZcEDb1lcA-1)

We could solve this as a part of #110 or if it will save us some time, only handle the accessibility statement. 

### Tasks

- [x] Design how it should look / Take #110 into concideration here
- [x] How do the app-developer configure this? handled by Studio team in this issue: https://github.com/Altinn/altinn-studio/issues/9585

### Acceptance Criterias
- [ ] A link is available from the footer 
- [ ] Default link is to our accessibility page
- [ ] The app developer should be able to overwrite this link and send their user to their own statement
